src/test/java/apicurito/tests/steps/DataTypeSteps.java

```diff
@@ -1,58 +1,65 @@
+            .filter(Condition.attribute("label", "PROPERTIES")).first();
+    private static SelenideElement INFO_SECTION = DataTypeUtils.getDataTypesRoot().$("definition-info-section");
     private static SelenideElement EXAMPLE_SECTION = DataTypeUtils.getDataTypesRoot().$("definition-example-section");
 
     @And("^create data type property with name \"([^\"]*)\"$")
     public void createDataTypeProperty(String property){
         CommonUtils.getClickableLink(CommonUtils.Sections.PROPERTIES, PROPERTIES_SECTION).click();
         CommonUtils.getLabelWithName("name", DataTypeUtils.getDataTypesRoot()).setValue(property);
-        CommonUtils.getButtonWithText("Add", DataTypeUtils.getDataTypesRoot()).click();
+        CommonUtils.getButtonWithText("Save", CommonUtils.getAppRoot().$("property-editor")).click();
     }
 
     @And("^set description \"([^\"]*)\" for property \"([^\"]*)\"$")
     public void setDescriptionForProperty(String description, String property) {
         DataTypeUtils.openPropertyDescription(property);
-        OperationUtils.clickToEditDescriptionTextArea(PROPERTIES_SECTION);
-        PROPERTIES_SECTION.$("ace-editor textarea").sendKeys(description);
-        CommonUtils.getButtonWithTitle("Save changes.", PROPERTIES_SECTION)
-                .click();
-    }
-
-    @And("^set type \"([^\"]*)\" formatted as \"([^\"]*)\" for property \"([^\"]*)\"$")
-    public void setTypeFormattedAsForProperty(String type, String as, String property) {
-        DataTypeUtils.openPropertyTypes(property);
-        DataTypeUtils.setPropertyType(type);
-        DataTypeUtils.setFormattedAs(as);
+        CommonUtils.setValueInTextArea(description, PROPERTIES_SECTION);
     }
 
     @And("^set data type description \"([^\"]*)\"$")
     public void setDataTypeDescription(String description) {
-        DataTypeUtils.clickToEditDescription(INFO_SECTION);
-        INFO_SECTION.$("ace-editor textarea").sendKeys(description);
-        CommonUtils.getButtonWithTitle("Save changes.", INFO_SECTION)
-                .click();
+        CommonUtils.setValueInTextArea(description, INFO_SECTION);
     }
 
     @And("^set data type example \"([^\"]*)\"$")
     public void setDataTypeExample(String example) {
-        DataTypeUtils.clickToEditDescription(EXAMPLE_SECTION);
-        EXAMPLE_SECTION.$("ace-editor textarea").sendKeys(example);
-        CommonUtils.getButtonWithTitle("Save changes.", EXAMPLE_SECTION)
-                .click();
+        CommonUtils.setValueInTextArea(example, EXAMPLE_SECTION);
     }
 
     @And("^set property \"([^\"]*)\" as required \"([^\"]*)\"$")
     public void setPropertyAsRequired(String property, Boolean isRequired) {
         DataTypeUtils.openPropertyTypes(property);
-        DataTypeUtils.setPropertyAsRequired(isRequired);
+        if(isRequired) {
+            CommonUtils.setDropDownValue(CommonUtils.DropdownButtons.REQUIRED.getButtonId(),"Required", PROPERTIES_SECTION);
```

Comment:
This could be rewritten using a ternary, the only difference between the branches is "Required" vs "Not requried"

Reply:
done